### PR TITLE
Update MessageBox.cfc

### PR DESCRIPTION
**Pull request by gpickin**

### CODE REVIEW

**Comment by gpickin** [comment]
What do you think about adding a traditional bootstrap view? 
Adding the options for primary, secondary and danger for those used to Bootstrap?

We can also add the normal cbmessagebox styles for those types as well, to the non bootstrap version.
And update the readme if you think its worth while

Also, it looks for the settings in the root of the coldbox struct, not in module settings, took me a minute or two to figure that out. 
Should we update that? Or make it look there too?

**Comment by gpickin** [comment]
If we want to add the styles for the default template, we could add this css

```
Primary:
Background is rgb(194,222,254)
Border is rgb(170,208,255)
Text is rgb(3,46,113)

Secondary:
Background is rgb(219,220,223)
Border is rgb(204,207,211)
Text is rgb(43,47,50)```

**Comment by lmajano** [comment]
@gpickin can you please update the readme file to document all the options and settings please.

**Comment by lmajano** [comment]
Also, if we move the settings to the `settings` thats a major bump.

**Comment by lmajano** [comment]
update on this @gpickin ?

**Comment by lmajano** [comment]
@gpickin let's revive this pr please, can you resolve the conflicts please first. I like the idea.

File: views/Bootstrap.cfm [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<cfsetting enablecfoutputonly=true>
```
Comment (lmajano) [inline views/Bootstrap.cfm:1]: How would you use this view then?

Comment (lmajano) [reply views/Bootstrap.cfm:1]: Can we get a tailwind one too?

**Review by lmajano — COMMENTED** [review]
>We can also add the normal cbmessagebox styles for those types as well, to the non bootstrap version.
YES Please.

> And update the readme if you think its worth while
YES Please

>Also, it looks for the settings in the root of the coldbox struct, not in module settings, took me a minute or two to figure that out.

I thought we updated this

**Comment by lmajano** [comment]
Ok, I think you sent this before I did the major update

**Comment by lmajano** [comment]
Please rebase and retry

**Pull request closed without merging**